rust-arroyo/src/backends/kafka/producer.rs

```diff
@@ -65,13 +68,21 @@ impl ClientContext for ProducerContext {
 }
 
 impl RdkafkaProducerContext for ProducerContext {
-    type DeliveryOpaque = ();
+    type DeliveryOpaque = Arc<Sender<Result<(), KafkaError>>>;
```

Comment:
why does this sender need to be arced?

Reply:
In order to match the type of DeliveryOpaque it needed to be Arc'd.